pkg/utils/strings.go

```diff
@@ -6,3 +6,12 @@ func IsValidUUID(u string) bool {
 	_, err := uuid.Parse(u)
 	return err == nil
 }
+
+func Contains(array []string, str string) bool {
```

Comment:
what about using the super intelligent `slices.contains` function instead ?

Reply:
I did it but this package no in go version `1.20` which is the version used by this repo.